components/Common/LanguageSelector/index.tsx

```diff
@@ -41,7 +41,7 @@ const LanguageSelector = () => {
   });
 
   return (
-    <>
+    <div ref={ref}>
```

Comment:
Please do not add a div here. We intentionally changed the Dropdown to contain the `ref` for obvious reasons.

Please undo this change. See details here: https://github.com/nodejs/nodejs.org/pull/5500

Reply:
(Please also add the ref back to the Dropdown), why did you remove the ref from there?

Reply:
> why did you remove the ref from there?

Well with the `ref` being set in the parent `div`, the `Dropdown` didn't need the `ref` as the same functionality could be done without passing it down.

Reply:
I still ask you, please, to revert this change. We don't want a pointless `div` surrounding the Language Selector button. Not to mention this is a regression to the bug mentioned before.

Please also refer to https://github.com/nodejs/nodejs.org/issues/5508

Reply:
> We don't want a pointless `div` surrounding the Language Selector button

this is the biggest argument to me, from a markup purity standpoint

Reply:
I've updated the changes to one where the DOM isn't changed.